setup.py

```diff
@@ -6,6 +6,9 @@
 )
 
 extras_require={
+    'ledger': [
+        "ledgerblue==0.1.17",
```

Comment:
Should we change this to be a version range, maybe just pinned within the `minor` version?  `ledgerblue>=0.1.17,<0.2.0`